tests/framework/extensions/crds/listcrd.go

```diff
@@ -0,0 +1,14 @@
+package crds
+
+import (
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+)
+
+func GetCRDs(steveClient *v1.Client, clusterID string) (*v1.SteveCollection, error) {
```

Comment:
Question did the existing one not work for you? https://github.com/rancher/rancher/blob/release/v2.7/tests/framework/extensions/kubeapi/customresourcedefinitions/list.go
Or you needed it to be a v1 Steve call?

Reply:
Based on the answer the previous question add a comment if we're keeping this.

Reply:
Wanted a steve call so added this extension. The other one is using dynamic client. So, added this extension.

Reply:
@igomez06 checked with devs on this and its preferable to use kubernetes API as with the public API the preferable direction is to use kubernetes API and steve endpoints are used by UI mostly and for these extensions they might not update to v1 endpoints yet. Will update the PR to use Kubernetes API.

Reply:
Spoke offline and it was agreed upon to remove clusterAuth altogether from rancher config and check if the cluster type if RKE1.